Header header
uint8 width
uint8 height
float32[] data
